stock_release_channel/models/res_partner.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    stock_release_channel_ids = fields.Many2many(
```

Comment:
@sbejaoui To be sure that the same table is used for each side of the many2many relation I would prefer to explicitely reference the relation, column1 and column2 paramters (id ResPartner and StockReleaseChannel)
```suggestion
    stock_release_channel_ids = fields.Many2many(
        comodel_name="stock.release.channel",
        relation="res_partner_stock_release_channel_rel",
        column1="partner_id",
        column2="channel_id",
        string="Release Channels"
    )
```